get_all_manifests.sh

```diff
@@ -6,20 +6,20 @@ GITHUB_URL="https://github.com/"
 # component: notebook, dsp, kserve, dashbaord, cf/ray/kueue/trainingoperator, trustyai, modelmesh, modelregistry.
 # in the format of "repo-org:repo-name:branch-name:source-folder:target-folder".
 declare -A COMPONENT_MANIFESTS=(
-    ["codeflare"]="opendatahub-io:codeflare-operator:main:config:codeflare"
-    ["ray"]="opendatahub-io:kuberay:dev:ray-operator/config:ray"
-    ["kueue"]="opendatahub-io:kueue:dev:config:kueue"
-    ["data-science-pipelines-operator"]="opendatahub-io:data-science-pipelines-operator:main:config:data-science-pipelines-operator"
-    ["odh-dashboard"]="opendatahub-io:odh-dashboard:main:manifests:dashboard"
-    ["kf-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/notebook-controller/config:odh-notebook-controller/kf-notebook-controller"
-    ["odh-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/odh-notebook-controller/config:odh-notebook-controller/odh-notebook-controller"
-    ["notebooks"]="opendatahub-io:notebooks:main:manifests:notebooks"
-    ["trustyai"]="trustyai-explainability:trustyai-service-operator:main:config:trustyai-service-operator"
-    ["model-mesh"]="opendatahub-io:modelmesh-serving:release-0.11.1:config:model-mesh"
-    ["odh-model-controller"]="opendatahub-io:odh-model-controller:release-0.12.0:config:odh-model-controller"
-    ["kserve"]="opendatahub-io:kserve:release-v0.11.1:config:kserve"
-    ["modelregistry"]="opendatahub-io:model-registry-operator:main:config:model-registry-operator"
-    ["trainingoperator"]="opendatahub-io:training-operator:dev:manifests:trainingoperator"
+    ["codeflare"]="opendatahub-io:codeflare-operator:v1.4.1-odh:config:codeflare"
+    ["ray"]="opendatahub-io:kuberay:v1.1.0-odh:ray-operator/config:ray"
+    ["kueue"]="opendatahub-io:kueue:v0.6.2-odh:config:kueue"
+    ["data-science-pipelines-operator"]="opendatahub-io:data-science-pipelines-operator:v2.2.0:config:data-science-pipelines-operator"
+    ["odh-dashboard"]="opendatahub-io:odh-dashboard:v2.22.2-odh-release:manifests:dashboard"
+    ["kf-notebook-controller"]="opendatahub-io:kubeflow:v1.7.0-9:components/notebook-controller/config:odh-notebook-controller/kf-notebook-controller"
+    ["odh-notebook-controller"]="opendatahub-io:kubeflow:v1.7.0-9:components/odh-notebook-controller/config:odh-notebook-controller/odh-notebook-controller"
+    ["notebooks"]="opendatahub-io:notebooks:v1.18.0:manifests:notebooks"
+    ["trustyai"]="trustyai-explainability:trustyai-service-operator:release/1.20.0:config:trustyai-service-operator"
```

Comment:
should this be 
["trustyai"]="trustyai-explainability:trustyai-service-operator:v1.20.0:config:trustyai-service-operator"
?

Reply:
Yes, but this is also a release branch, so I think it is okay for now. We can update it in next release